src/ui/components/search/searchcomponent.js

```diff
@@ -314,7 +314,7 @@ export default class SearchComponent extends Component {
     return 'search/search';
   }
 
-  onCreate () {
+  afterAnswersOnReady () {
```

Comment:
This is 🥇 . The change I never knew I needed

Reply:
talked with Tom and we ended up killing it :( The idea is that only the searchbar (and filtersearch ~kind of) need this functionality, so for now it's just a thing that the searchbar can do

Reply:
we can probably refactor these two components to not be responsible for searching on page load, though it would probably be difficult, but the we wouldn't need this lifecycle hook